src/js/App/Header/UserToggle.js

```diff
@@ -51,7 +51,7 @@ function buildItems(username, isOrgAdmin, accountNumber = -1, isInternal, extraI
       {accountNumber > -1 && (
         <DropdownItem
           component={
-            <ChromeLink href="/user-preferences/email" isBeta={isBeta()} appId="userPreferences">
+            <ChromeLink href={`/user-preferences/email#bundle=${getUrl('bundle')}&appId=${appId}`} isBeta={isBeta()} appId="userPreferences">
```

Comment:
I think we want to exclude the bundle if its `undefined` that can happen on landing page.